coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -264,6 +264,32 @@ def test_func_dummy(context, inputs):
         # Cleanup the test
         del _TORCH_OPS_REGISTRY.name_to_func_mapping["test_func_dummy"]
 
+    @staticmethod
+    def test_is_torch_fx_node_supported():
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, a, x, b):
+                y = torch.mm(a, x)
+                z = y + b
+                a = z - a
+                y = torch.mm(a, x)
+                z = y + b
+                return z
+            
+        model = Model()
+        model.eval()
+
+        symbolic_traced = torch.fx.symbolic_trace(model)
+
+        for node in symbolic_traced.graph.nodes:
+            if node.op == "call_function":
```

Comment:
Could you add some details here. I'm guessing it's `torch.mm` that we don't support but we support everything else.

Reply:
Added comments to explain
```
        for node in symbolic_traced.graph.nodes:
            # There are 5 types of torch fx node:
            #     1. placeholder
            #     2. get_attr
            #     3. call_function
            #     4. call_module
            #     5. output
            # Only "call_function" contains a single PyTorch op that can be queried in _TORCH_OPS_REGISTRY
            #
            # All PyTorch ops in the example model are supported, so they should all return true
            if node.op == "call_function":
                assert ct.utils.is_torch_fx_node_supported(node)
            # Other types of torch fx node are not supported
            else:
                assert not ct.utils.is_torch_fx_node_supported(node)
```